conjur/controller/init_controller.py

```diff
@@ -97,7 +98,8 @@ def validate_conjur_url(self, conjur_url: ParseResult):
         Raises a RuntimeError in case of an invalid url format
         """
         if conjur_url.scheme != 'https':
-            raise RuntimeError(f"Error: undefined behavior. Reason: The Conjur URL format provided "
+            raise InvalidURLFormatException(f"Error: undefined behavior. "
+                               f" Reason: The Conjur URL format provided "
                                f"'{self.conjurrc_data.conjur_url}' is not supported.")
```

Comment:
Should there be a `.` at the end of `provided`?
```suggestion
            raise InvalidURLFormatException(f"Error: undefined behavior. "
                               f" Reason: The Conjur URL format provided. "
                               f"'{self.conjurrc_data.conjur_url}' is not supported.")
```

Reply:
done